rollup.config.js

```diff
@@ -25,8 +25,7 @@ function createEntry(options) {
     input,
     external: [
       'vue',
-      'lodash/mergeWith',
-      'lodash/isString'
+      '@vue/compiler-dom'
```

Comment:
Not needed in master right now, but will be very soon for [this feature](https://github.com/vuejs/vue-test-utils-next/pull/115) when merged, so it's fine to include it here.